packages/core/src/index.ts

```diff
@@ -10,3 +10,6 @@ export {
   HuggingFaceEmbedding,
   HuggingFaceEmbeddingModelType,
 } from "./embeddings/HuggingFaceEmbedding.js";
+
+export { type VertexGeminiSessionOptions } from "./llm/gemini/types.js";
```

Comment:
@parhammmm the exports that you add here are only used for NodeJS - so this change means VertexGemini does only work with NodeJS (not edge for example) - if that' ok, nothing to do - otherwise you can send a PR and move this to `src/llm/index.ts`